Samples/iOS-Swift/iOS-Swift/Tools/SpanObserver.swift

```diff
@@ -35,13 +35,13 @@ class SpanObserver: NSObject {
     func addSpanObserver(forKeyPath keyPath: String, callback : @escaping (Span) -> Void) {
         callbacks[keyPath] = callback
         //The given span may be a SentryTracer that wont respond to KVO. We need to get the root Span
-        let span = span.rootSpan() ?? span
-        (span as? NSObject)?.addObserver(self, forKeyPath: keyPath, options: .new, context: nil)
+        let spanToObserve = span.rootSpan() ?? span
```

Comment:
Didn't compile with Xcode 12.

Reply:
Do you have the error message?

Reply:
Here you go https://github.com/getsentry/sentry-cocoa/runs/4559714235?check_suite_focus=true